stl/inc/algorithm

```diff
@@ -1789,9 +1789,9 @@ namespace ranges {
     template <bidirectional_iterator _It1, bidirectional_iterator _It2>
         requires indirectly_movable<_It1, _It2>
     _NODISCARD constexpr _It2 _Move_backward_common(const _It1 _First, _It1 _Last, _It2 _Result) {
-        if constexpr (_Ptr_move_cat<_It1, _It2>::_Trivially_copyable) {
+        if constexpr (_Memmove_in_move_is_safe<_It1, _It2>) {
             if (!_STD is_constant_evaluated()) {
-                return _Copy_backward_memmove(_First, _Last, _Result);
+                return _Memmove_backward(_First, _Last, _Result);

```

Comment:
Not really, The reason I have them is because of all the renaming of the mem-ops.

I had the hope that this would be merged sometime in the near future and wanted to avoid merge conflicts. Shout if you want to remove those changes

Reply:
I, for one, would prefer for these changes to be removed - they make it harder to review this PR independently.

Reply:
Removed :(